packages/in_app_purchase/in_app_purchase_android/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
-## 0.1.4+7
```

Comment:
This was published; it shouldn't be removed from the changelog.

Reply:
You are right and fixed it.